roles/nodejs/defaults/main.yml

```diff
@@ -1,3 +1,3 @@
 ---
 # nodejs version to install
-nodejs_major_version: "4"
+nodejs_major_version: "6"
```

Comment:
@armab When we do this, we probably need to a ansible-st2 release. I am not sure if this is the best way to do it or let people override this version in their playbooks. I am open to suggestions.